cider-velox/src/CiderJoinBuild.cpp

```diff
@@ -25,7 +25,7 @@
 
 namespace facebook::velox::plugin {
 
-void CiderJoinBridge::setData(std::vector<RowVectorPtr> data) {
+void CiderJoinBridge::setData(std::vector<VectorPtr> data) {
```

Comment:
is this your code change? seems curious.

Reply:
That is rebase code.